plugins/cue_points/quiz/services/QuizService.php

```diff
@@ -184,7 +186,11 @@ public function getUrlAction($entryId)
 		if ( is_null( $kQuiz ) )
 			throw new KalturaAPIException(KalturaQuizErrors::PROVIDED_ENTRY_IS_NOT_A_QUIZ, $entryId);
 
-		$finalPath ='/api_v3/index.php/service/quiz_quiz/action/serve/entryId/';
+		if ($quizFileType != KalturaQuizFileType::PDF)
+		{
+			throw new KalturaAPIException(KalturaQuizErrors::NO_SUCH_FILE_TYPE);
+		}
+		$finalPath ='/api_v3/index.php/service/quiz_quiz/action/serve/quizFileType/1/entryId/';
```

Comment:
Please change to /api_v3/service/.. and use constant instead of hard-coded output type.